modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/dashboard-collection/fragments/partner-level-dashlet/index.js

```diff
@@ -9,226 +9,214 @@
-				</span>
-
-				{text && <span className="col text-paragraph">{text}</span>}
-			</div>
-
-			{children && <div className="col">{children}</div>}
-		</div>
-	);
-};
-
-const levelProperties = {
-	Gold: {
-		arr: {
-			arr: 125000,
-			npOrNb: 2,
-		},
-		headcount: {
-			marketing: 1,
-			sales: 3,
-		},
-	},
-	Platinum: {
-		arr: {
-			arr: 250000,
-		},
-		headcount: {
-			marketing: 1,
-			sales: 5,
-		},
-	},
-	Silver: {
-		headcount: {
-			marketing: 1,
-			sales: 1,
-		},
-	},
-};
-
 export default function () {
-	const [data] = useState(partnerLevelData);
+	const [data, setData] = useState({});
+	const [headcount, setHeadcount] = useState({});
 	const [completed, setCompleted] = useState({});
-	const level = PartnershipLevels.GOLD;
-
-	useEffect(() => {
-		const checkedItems = {};
-
-		if (level !== PartnershipLevels.AUTHORIZED) {
-			if (data.certification) {
-				checkedItems['certification'] = true;
-			}
-
-			if (
-				data.headcount.marketing ===
-					levelProperties[level].headcount.marketing &&
-				data.headcount.sales === levelProperties[level].headcount.sales
-			) {
-				checkedItems['headcount'] = true;
+	const [loading, setLoading] = useState(false);
+
+	const getAccountInformation = async () => {
+		setLoading(true);
+
+		// eslint-disable-next-line @liferay/portal/no-global-fetch
+		const myUserAccountsRequest = await fetch(
+			'/o/headless-admin-user/v1.0/my-user-account',
+			{
+				headers: {
+					'accept': 'application/json',
+					'x-csrf-token': Liferay.authToken,
+				},
 			}
+		);
+
+		if (myUserAccountsRequest.ok) {
+			const {accountBriefs} = await myUserAccountsRequest.json();
+
+			if (accountBriefs.length) {
+				// eslint-disable-next-line @liferay/portal/no-global-fetch
+				const accountRequest = await fetch(
+					`/o/headless-admin-user/v1.0/accounts/${accountBriefs[0].id}`,
+					{
+						headers: {
+							'accept': 'application/json',
+							'x-csrf-token': Liferay.authToken,
+						},
+					}
+				);
+
+				// eslint-disable-next-line @liferay/portal/no-global-fetch
+				const accountUsersRequest = await fetch(
+					`/o/headless-admin-user/v1.0/accounts/${accountBriefs[0].id}/user-accounts`,
+					{
+						headers: {
+							'accept': 'application/json',
+							'x-csrf-token': Liferay.authToken,
+						},
+					}
+				);
+
+				const checkedItems = {};
+
+				if (accountRequest.ok) {
+					const accountData = await accountRequest.json();
+
+					if (
+						accountData.partnerLevel !==
+						PartnershipLevels.AUTHORIZED
+					) {
+						if (accountData.solutionDeliveryCertification) {
+							checkedItems[
+								'solutionDeliveryCertification'
+							] = true;
+						}
 
-			if (level !== PartnershipLevels.SILVER) {
-				if (data.marketing) {
-					checkedItems['marketing'] = true;
-				}
+						if (
+							accountData.partnerLevel !==
+							PartnershipLevels.SILVER
+						) {
+							if (accountData.marketingPlan) {
+								checkedItems['marketingPlan'] = true;
+							}
```

Comment:
instead we have a if with a true assignment, you can pass the verification directly.